src/views/task-list-page/task-list-workers-table/task-list-workers-table.tsx

```diff
@@ -0,0 +1,60 @@
+'use client';
+import React from 'react';
+
+import Table from '@/components/table/table';
+import usePageQueryParams from '@/hooks/use-page-query-params/use-page-query-params';
+import sortBy, { toggleSortOrder } from '@/utils/sort-by';
+
+import taskListPageQueryParamsConfig from '../config/task-list-page-query-params.config';
+import taskListWorkersTableConfig from '../config/task-list-workers-table.config';
+
+import filterWorkers from './helpers/filter-workers';
+import isValidTableColumn from './helpers/is-valid-table-column';
+import { styled } from './task-list-workers-table.styles';
+import { type Props } from './task-list-workers-table.types';
+
+export default function TaskListWorkersTable({ taskList }: Props) {
+  const [queryParams, setQueryParams] = usePageQueryParams(
+    taskListPageQueryParamsConfig
+  );
+
+  const filteredWorkers = filterWorkers({
+    workers: taskList.workers,
+    handlerType: queryParams.handlerType,
+    search: queryParams.taskListSearch,
+  });
+
+  return (
+    <styled.TableContainer>
+      <Table
+        data={sortBy(
+          filteredWorkers,
+          (w) =>
+            isValidTableColumn(queryParams.sortColumn)
```

Comment:
it can be much better to do this from page query params parse function to make sure that the value is consistent across the whole code branches

Reply:
I'll move the isValidTableColumn type guard to parseValue